src/modules/extractors/items.extractor.ts

```diff
@@ -157,11 +175,54 @@ export class ItemsExtractor {
             this.itemNameMap.set(item.name, item);
           }
         }
+
+        item.aliases.forEach((alias) => {
+          if (!alias) return;
+          // Not really sure why, but some aliases are missing the closing parenthesis
+          const unterminated = this.hasUnterminatedParenthesis(alias);
+          if (!this.aliasNameMap.has(alias)) {
+            this.aliasNameMap.set(alias, item);
+            if (unterminated) {
+              this.aliasNameMap.set(alias + ')', item);
+            }
+          } else {
+            const otherItem = this.aliasNameMap.get(alias);
+            // Score depending on the amount of "true", with priority to being in the main game
+            const score =
+              Number(item.isInMainGame) * 3 +
+              Number(item.isOnGrandExchange) +
+              Number(item.isTradeable);
+            const otherScore =
+              Number(otherItem.isInMainGame) * 3 +
+              Number(otherItem.isOnGrandExchange) +
+              Number(otherItem.isTradeable);
```

Comment:
As this is getting repeated few additional times, I'd extract it in a `weightItemScore` method so it's easier to tweak in the long run